frontend/components/exercise/AudioSelectionExercise.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
```

Comment:
overall comment:
go through and change any padding/margins/gaps to make sure they have [arbitrary values px]